pkgs/games/teeworlds/default.nix

```
@@ -1,4 +1,4 @@
-{ fetchFromGitHub, lib, stdenv, cmake, pkg-config, python3, alsa-lib
+{ fetchFromGitHub, fetchpatch, lib, stdenv, cmake, pkg-config, python3, alsa-lib
```

```suggestion
{ fetchFromGitHub, fetchpatch2, lib, stdenv, cmake, pkg-config, python3, alsa-lib
```
`fetchpatch2` uses a newer version of `patchutils`, so I feel we should use that instead.

---

I'm not against it but the vast majority of recent changes in nixpkgs still use `fetchpatch`, we do not need any of the features that comes with `fetchpatch2` (and it appears the initial motivation to introduce it were not resolved https://github.com/NixOS/nixpkgs/issues/32084) so I'm not really sure if it is really worth it at this point.

---

Fair enough.